### ISSUE TITLE
How many mAP your model can improve?

### ISSUE THREAD

**Issue by Changes92** [Lechatelia/faster-rcnn-with-gcn#2]
I really like your work.

Could you tell me what dataset is used in your project and how many mAP can you improve? Thank you.

**Comment by Lechatelia** [comment]
I used the VOC and COCO2014 as the training dataset. Maybe the COCO2017 is also ok with small changes.
Sorry, I didn't have extra time to tune the hyper parameters in the GCN method, so i only got improvement less than 0.5 mAP (i forgot the specific mAP result).

By the way, it is easy to be overfitting for GCN, so the dropout layer is crucial.
